containers/unit_tests/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ foreach(Tag Threads;Serial;OpenMP;HPX;Cuda;HIP;SYCL)
       DynViewAPI_generic
       DynViewAPI_rank12345
       DynViewAPI_rank67
+      DynRankView_TeamScratch
```

Comment:
This naming scheme somewhat clashes with all the other existing tests in this list...

Reply:
I would want to rename the other ones, since its confusing whether DynView means DynamicView or DynRankView